composer.json

```diff
@@ -31,12 +31,12 @@
 		"php": ">=5.6.0"
 	},
 	"require-dev": {
-		"phpcompatibility/phpcompatibility-wp": "^2.1.0",
+		"phpcompatibility/phpcompatibility-wp": "^2.1.3",
 		"dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
-		"squizlabs/php_codesniffer" : "^3.5.4",
-		"wp-coding-standards/wpcs": "*",
+		"squizlabs/php_codesniffer" : "^3.6.0",
+		"wp-coding-standards/wpcs": "~2.3.0",
 		"wp-phpunit/wp-phpunit": "^5.9",
-		"phpunit/phpunit": "^7.5",
+		"phpunit/phpunit": "^9.0",
```

Comment:
This is blocking WordPress 5.7 to be tested properly in the Github Action CI (we don't see the failed actions because they are not active for pull requests). We need to install the `"phpunit/phpunit": "^7.5"` version in the Github Action only for WordPress 5.7 or previous WordPress versions (right now, we are only testing 5.7)

Reply:
Ok, I've removed the line completely as you suggested in the previous comment.